### PR TITLE
#165412913 sign out a user

### PR DESCRIPTION
**Pull request by rwajon**
#### What does this PR do?
Sign out a user from the application
#### Description of Task to be completed?
Blacklist a token to make sure it is invalid
#### How should this be manually tested?
- After cloning the repo, cd into it and type `npm test` in the CLI
- Using postman, run `npm run dev`, then login first by typing `http://localhost:<PORT>/auth/login`, after this, copy the received token and type `http://localhost:<PORT>/auth/logout` then in the header pass the token with the following key name: `access-token`
#### What are the relevant pivotal tracker stories?
[165412913](https://www.pivotaltracker.com/story/show/165412913)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,11 +23,9 @@
     "@hapi/joi": "^15.0.0",
     "D": "^1.0.0",
```
Comment (djallas) [inline package.json]: What does this package do?

File: src/migrations/20190428161016-create-comment.js [hunk 2]
```diff
@@ -11,24 +11,24 @@ export default {
...
+        key: 'id'
       },
       onUpdate: 'CASCADE',
-      onDelete: 'SET NULL',
+      onDelete: 'SET NULL'
     },
     body: {
       type: Sequelize.TEXT,
-      allowNull: false,
+      allowNull: false
```
Comment (sengayire) [inline src/migrations/20190428161016-create-comment.js]: I can see you allowed the body of comment to be null, is it good to allow someone to create a comment with null body?

File: src/migrations/20190509005606-create-token.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Tokens', {
+    id: {
```
Comment (sengayire) [inline src/migrations/20190509005606-create-token.js]: do we really need to create a table for a token, if yes why

Comment (djallas) [reply src/migrations/20190509005606-create-token.js]: I think so. Because we need to blacklist invalid token

File: package.json (hunk 1 above)
Comment (rwajon) [reply package.json]: this package was installed as a dependency to generate random data

**Pull request merged**